### PR TITLE
fix: add doc  #834

### PR DESCRIPTION
**Pull request by DaiQiangReal**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [x] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- Documentation: 规范 TagGroup tagList props 类型

---

🇺🇸 English
- Documentation: Specification TagGroup tagList props type

### Checklist
- [ ] Test or no need
- [X] Document or no need
- [ ] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

**Comment by shijiatongxue** [comment]
image

**Comment by DaiQiangReal** [comment]
> build 失败了
> 
> image

看到了，刚发现 treeSelect 里也有 tag 这种用法

File: packages/semi-ui/tag/group.tsx [hunk 1]
```diff
@@ -10,20 +10,20 @@ const prefixCls = cssClasses.PREFIX;
...
 const avatarShapeSet = strings.AVATAR_SHAPE;
 
-export interface TagGroupProps {
+export interface TagGroupProps<T> {
     style?: React.CSSProperties;
     className?: string;
     maxTagCount?: number;
     restCount?: number;
-    tagList?: (TagProps | React.ReactNode)[];
+    tagList?: (T extends 'custom' ? (TagProps | React.ReactNode) : TagProps)[];
```
Comment (shijiatongxue) [inline packages/semi-ui/tag/group.tsx:18]: custom 时，tagList item 可以是 TagProps 吗？

Comment (DaiQiangReal) [reply packages/semi-ui/tag/group.tsx]: 改了

**Pull request merged**